tests/categorical/test_contingency.py

```diff
@@ -368,3 +377,19 @@ def test_dask_if_available_categorical():
 
     # And that transformed things get the same numbers
     assert table.false_alarm_rate() == table.transform().false_alarm_rate()
+
+
+def test_heidke_table():
+    """
+    Test the Heidke Skill Score with a known example
+    """
+
+    table = finlayson_table
+    cm = scores.categorical.BasicContingencyManager(table)
+    heidke = cm.heidke_skill_score()
+
+    # Note - the reference in the CAWCR verification site has 0.36 for
+    # the expected result, but presumably this is rounded to two decimal
+    # places
+    expected = xr.DataArray(0.355325)
```

Comment:
@nicholasloveday, please check this result is expected given the CAWCR site lists 0.36.